pillar.example

```diff
@@ -1,46 +1,94 @@
+---
+# If no configuration is set, a default one
+#  (potentially wrong and insecure) is generated
 squid:
-  example_rule: True
-
+  # listen to HTTP on these ports
   http_port:
      - 3128
-     - 127.0.0.1:8080
+     - 8080
```

Comment:
In fact... The commit is from several months ago, so I don't remember why I did this, maybe to match the default configuration, but I'm not sure. Maybe it's just 'stupid'.